frontend/app/dashboard/create-a-team/page.jsx

```diff
@@ -0,0 +1,10 @@
+const CreateTeam = () => {
```

Comment:
*folder structure

Its a little weird having a page folder nested inside another page folder. Would this folder be better placed one level. up? Or alternatively maybe in the components folder.